### DISCUSSION TITLE
MacOS 11.1

### DISCUSSION THREAD

**Discussion by lpke** [asmvik/yabai#770] [Q&A] [answered]
Hi,

Apologies if this has already been asked but I couldn't find anything on it.

Does Yabai have any issues running on MacOS 11.1? I have the update ready but have been putting off installing until I'm sure Yabai will work fine on it (including all SA features).

**Comment by rockyzhang24** [comment, marked as answer]
yabai is running perfectly on my machine.
* 16 inch MacBook Pro with macOS 11.1.
* yabai: the latest master (installed with `HEAD`)

**Comment by lpke** [comment]
> yabai is running perfectly on my machine.
> 
>     * 16 inch MacBook Pro with macOS 11.1.
> 
>     * yabai: the latest master (installed with `HEAD`)

Happy to hear it. I don't currently have Yabai installed from `HEAD` though, so I might look into that if this thread doesn't get any other replies from people running release versions. I'll likely update and see how I go, and fall back to `HEAD` if need be. When I update I'll report back here with my experience.

**Comment by rockyzhang24** [comment]
The released version worked well on my machine as well. I updated yabai to the newest master from the released version (which is the second newest commit on the master actually) this morning.

**Comment by dominiklohmann** [comment]
Heads up: I converted this into a discussion and selected the first answer to be the correct one, as this is not an issue.
